src/main/pages/che-7/end-user-guide/proc_defining-maven-settings-xml-file-across-workspaces.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// using-maven-artifact-repositories
+
+[id="proc_defining-maven-settings-xml-file-across-workspaces_{context}"]
+= Defining maven `settings.xml` file across workspaces
+
+To specify your own `settings.xml` file across workspaces, use the Secret with `settings.xml` and define labels and annotation to mount it as a file (see link:{site-baseurl}che-7/mounting-a-secret-as-a-file-or-an-environment-variable-into-a-workspace-container/#mounting-a-secret-as-a-file-into-a-workspace-container_mounting-a-secret-as-a-file-or-an-environment-variable-into-a-workspace-container[Mounting a secret as a file into a workspace container]).
```

Comment:
```suggestion
To specify your own `settings.xml` file across workspaces, use the `Secret` kind in `settings.xml` and define labels and annotation to mount it as a file (see link:{site-baseurl}che-7/mounting-a-secret-as-a-file-or-an-environment-variable-into-a-workspace-container/#mounting-a-secret-as-a-file-into-a-workspace-container_mounting-a-secret-as-a-file-or-an-environment-variable-into-a-workspace-container[Mounting a secret as a file into a workspace container]).
```

Reply:
Settings.xml is content of Secret. IMO your suggestion is not technically correct.

Reply:
I thought it referred to this line in in the yaml:
```
kind: Secret
```
If not, please, qualify 'Secret' in this context.

Reply:
Let me try to rephrase then, because the current wording is IMHO a bit condensed:

To use your own `settings.xml` file across all your workspaces, you can create a `Secret` object (with a name of your choice) in the same namespace as the workspace. Put the contents of the required `settings.xml` in the data section of the `Secret` (possibly along with other files that should reside in the same directory as it). Labelling and annotating this `Secret` according to link:{site-baseurl}che-7/mounting-a-secret-as-a-file-or-an-environment-variable-into-a-workspace-container/#mounting-a-secret-as-a-file-into-a-workspace-container_mounting-a-secret-as-a-file-or-an-environment-variable-into-a-workspace-container[this guide] will make sure that the contents of the secret are mounted into the workspace pod. Note that you need to restart any previously running workspaces for them to use this secret.

 Does that sound a little bit clearer to you, @rkratky ?

Reply:
without your permission, I've copy-pasted it :) thanks @metlos